config/config.exs

```diff
@@ -19,7 +19,7 @@ config :identity_web,
 
 # Configures the endpoint
 config :identity_web, IdentityWeb.Endpoint,
-  url: [host: "localhost"],
+  url: [host: "localhost", port: System.get_env("PORT", "4000")],
```

Comment:
Are you sure about the port here ?
```suggestion
  url: [host: "localhost", port: System.get_env("PORT", "4010")],
```

Reply:
Missed it & env variable is wrong too ! Thx :)